README.md

```diff
@@ -305,4 +305,4 @@ git push
 * [Developer Docs](https://developer.tbd.website/docs/web5/)
 * [Coding Conventions](./CONVENTIONS.md)
 * [Code of Conduct](./CODE_OF_CONDUCT.md)
-* [Governance](./GOVERNANCE.md)
\ No newline at end of file
+* [Governance](./GOVERNANCE.md)
```

Comment:
trying to figure out what changed here lol

Reply:
new line ? (but normally it tells us I thought)

Sorry this pr wasn't for actually updating the readme, I thought that mainline was borked, but actually we are good to go